wire-ios/Wire-iOS Tests/ConversationMessageCell/MessageToolboxViewTests.swift

```diff
@@ -49,7 +50,22 @@ final class MessageToolboxViewTests: CoreDataSnapshotTestCase {
         sut.configureForMessage(message, forceShowTimestamp: true, animated: false)
 
         // THEN
-        verify(view: sut)
+        verifyInAllPhoneWidths(view: sut)
+    }
+
+    func testThatItConfiguresWithFailedToSendAndReason() {
+        // GIVEN
+        message.deliveryState = .failedToSend
+        message.conversationLike = otherUserConversation
+        message.failedToSendReason = .federationRemoteError
+        message.conversation?.domain = "anta.wire.link"
+    
+
+        // WHEN
+        sut.configureForMessage(message, forceShowTimestamp: true, animated: false)
+
+        // THEN
+        verifyInAllPhoneWidths(view: sut)
```

Comment:
Is it necessary to have a snapshot for every phone width?

Reply:
It just seems a bit wasteful to have so many snapshot tests. If you want to test varying widths, perhaps it's sufficient to test a narrow and a wide width.

Reply:
Updated snapshot tests.